README.md

```diff
@@ -1,3 +1,5 @@
+This project is forked from [jiffyclub/html-css-resume-template](https://github.com/jiffyclub/html-css-resume-template).
```

Comment:
Could you take this commit out? This would show up in my repo too. (I think github's annotation of where things are forked from is usually enough to point people back to the source.)